owncloudComLibrary/src/test/java/com/owncloud/android/lib/ShareeResponseTest.kt

```diff
@@ -0,0 +1,145 @@
+package com.owncloud.android.lib
+
+import com.owncloud.android.lib.resources.CommonOcsResponse
+import com.owncloud.android.lib.resources.shares.responses.ShareeOcsResponse
+import com.squareup.moshi.JsonAdapter
+import com.squareup.moshi.Moshi
+import com.squareup.moshi.Types
+import junit.framework.Assert.assertEquals
+import junit.framework.Assert.assertTrue
+import org.junit.Assert.assertNotEquals
+import org.junit.Before
+import org.junit.Test
+import java.lang.reflect.Type
+
+class ShareeResponseTest {
+
+    var response: CommonOcsResponse<ShareeOcsResponse>? = null
+
+    @Before
+    fun prepare() {
+        val moshi = Moshi.Builder().build()
+        val type: Type = Types.newParameterizedType(CommonOcsResponse::class.java, ShareeOcsResponse::class.java)
+        val adapter: JsonAdapter<CommonOcsResponse<ShareeOcsResponse>> = moshi.adapter(type)
+        response = adapter.fromJson(EXAMPLE_RESPONSE)
```

Comment:
`response` will be the same response always. Maybe we should also check edge cases, such as empty responses etc. What do you think?

BTW, add license header